ssm.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_ssm_parameter" "namespace_id" {
```

Comment:
if you add more resources that depend on `count` I would recommend to move them to a new module so the count is at the module level

Reply:
That would be lovely if all SSM params had the same condition in count.